app/controllers/openid_connect/logout_controller.rb

```diff
@@ -61,6 +61,7 @@ def handle_successful_logout_request(result, redirect_uri)
           post_logout_redirect_uri: logout_params[:post_logout_redirect_uri],
         }
         @params[:state] = logout_params[:state] if !logout_params[:state].nil?
+        @service_provider_name = @logout_form.service_provider&.friendly_name
```

Comment:
Is it even possible to get to this point without `service_provider` being present?

Reply:
Currently no, but we have some potential changes on the horizon where that may change.